README.md

```diff
@@ -259,7 +260,7 @@ See [CONTRIBUTING.md](./docs/CONTRIBUTING.md).
   [-d <RTL-SDR USB device index>] (default: 0)
   [-d :<RTL-SDR USB device serial (can be set with rtl_eeprom -s)>]
 	To set gain for RTL-SDR use -g <gain> to set an overall gain in dB.
-	SoapySDR device driver is available.
+	SoapySDR device driver is not available.
```

Comment:
?

Reply:
This is a one of update when I run maintainer_update.py. I have no idea. Caused by my environment?

Reply:
reverted.